src/ios/CDVCamera.m

```diff
@@ -396,13 +396,16 @@ - (NSString*)tempFilePath:(NSString*)extension
     NSString* docsPath = [NSTemporaryDirectory()stringByStandardizingPath];
     NSFileManager* fileMgr = [[NSFileManager alloc] init]; // recommended by Apple (vs [NSFileManager defaultManager]) to be threadsafe
     NSString* filePath;
-
+    
     // generate unique file name
-    int i = 1;
+    //int i = 1; 
+	NSTimeInterval timeStamp = [[NSDate date] timeIntervalSince1970];
```

Comment:
Hi @janpio, I see you already fixed indention and commit in `klodha:klodha-camera-ios-patch`. So is there any further change required from my side?

Reply:
Thanks. I really hope if someone get this merged into main branch. Because this particular problem of getting old images is really hard to debug (at least it was for me), and for some time I couldn't figure at all why it is happening. 

Hopefully getting this fix as a part of core plugin, will save other new users from trouble I faced.